AB#1167 file encoding (#25)
edgelesssys/docs.edgeless.systems: marblerun/workflows/define-manifest.md

* By using a direct mapping of filename to content: `"<FileName>": "<Content>"`
* By specifying an encoding for the content, and optionally, if the content contains templates:
    ```javascript
    "<FileName>": {
        "Data": "<Content>",
        "Encoding": "<EncodingType>",
        "NoTemplates": true/false
    }
    ```
    * `Data`: The file content
    * `Encoding`: Allows users to encode the `Data` field of the manifest. Marbles receive the decoded value. The following options are available:
        * `string`: No encoding. This is the default
        * `base64`: The manifest contains `Data` in [Base64](https://pkg.go.dev/encoding/base64). This can be useful to set content that can otherwise not be parsed in JSON format, or to avoid having to worry about correctly escaping newlines in a multiline document
        * `hex`: Same as `base64`, but [Hex Encoding](https://pkg.go.dev/encoding/hex) is used instead
    * `NoTemplates`: If this flag is set, content in `Data` is not processed for templates. Use this if your file contains [Go Templates](https://golang.org/pkg/text/template/) structures that should not be interpreted by MarbleRun.

> 

Reviewer: Please add something for `Argv` as well.

>     * `NoTemplates`: If this flag is set, content in `Data` is not processed for templates. Use this if your file needs to contain strcutures similar to those used in [Go Templates](https://golang.org/pkg/text/template/), or as a hint to other users that an encoded file declaration does not contain additional templates.

Reviewer: ```suggestion
    * `NoTemplates`: If this flag is set, content in `Data` is not processed for templates. Use this if your file contains [Go Templates](https://golang.org/pkg/text/template/) structures that should not be interpreted by MarbleRun.
```
